include/openssl/ssl.h.in

```diff
@@ -316,158 +316,135 @@ typedef int (*SSL_verify_cb)(int preverify_ok, X509_STORE_CTX *x509_ctx);
 /* Typedef for SSL async callback */
 typedef int (*SSL_async_callback_fn)(SSL *s, void *arg);
 
-/* Disable Extended master secret */
-# define SSL_OP_NO_EXTENDED_MASTER_SECRET                0x00000001U
 
-/* Cleanse plaintext copies of data delivered to the application */
+/*
+ * SSL/TLS Connection Options.
+ */
+    /* Disable Extended master secret */
+# define SSL_OP_NO_EXTENDED_MASTER_SECRET                0x00000001U
+    /* Cleanse plaintext copies of data delivered to the application */
 # define SSL_OP_CLEANSE_PLAINTEXT                        0x00000002U
-
-/* Allow initial connection to servers that don't support RI */
+    /* Allow initial connection to servers that don't support RI */
 # define SSL_OP_LEGACY_SERVER_CONNECT                    0x00000004U
-
-/* Enable support for Kernel TLS */
+    /* Enable support for Kernel TLS */
 # define SSL_OP_ENABLE_KTLS                              0x00000008U
-
 # define SSL_OP_TLSEXT_PADDING                           0x00000010U
+# define SSL_OP__UNUXED                                  0x00000020U
```

Comment:
I think this should just be a comment noting that it is unused. Adding a special define means that people might start relying on it being there. If we end up using a `uint64_t` for the OP value then there will be lots of unused slots and it doesn't seem sensible to have a define for each one.